github/repos.go

```diff
@@ -538,6 +540,12 @@ type RequiredPullRequestReviews struct {
 	IncludeAdmins bool `json:"include_admins"`
```

Comment:
@shurcooL No reply from support yet but here are test results:
1. When required_pull_request_reviews is not in the request, response status is 200 OK - 
existing required_pull_request_reviews is not affected
1. When required_pull_request_reviews is null, response is 200 OK - existing required_pull_request_reviews is removed from the resource
1. When required_pull_request_reviews is {}, response is 422 Unprocessable Entity, with following error in body:
```
{
  "message": "Invalid request.\n\nNo subschema in \"anyOf\" matched.\n\"include_admins\" wasn't supplied.\nNot all subschemas of \"allOf\" matched.\nFor 'anyOf/1', {} is not a null.",
  "documentation_url": "https://developer.github.com/v3/repos/branches/#update-branch-protection"
}
```
So, it's safe to remove required_pull_request_reviews from the request completely. 

For required_status_checks:
1. When required_status_checks is sent with include_admins, , response is 200 OK - existing required_status_checks is updated with new values
1. When required_status_checks is sent without include_admins, response is 422 Unprocessable Entity, with following error in body:
```
{
  "message": "Invalid request.\n\nNo subschema in \"anyOf\" matched.\n\"include_admins\" wasn't supplied.\nNot all subschemas of \"allOf\" matched.\nFor 'anyOf/1', {\"url\"=>\"https://api.github.com/repos/amirilovic/test-repo/branches/master/protection/required_status_checks\", \"strict\"=>true, \"contexts\"=>[], \"contexts_url\"=>\"https://api.github.com/repos/amirilovic/test-repo/branches/master/protection/required_status_checks/contexts\"} is not a null.",
  "documentation_url": "https://developer.github.com/v3/repos/branches/#update-branch-protection"
}
```

So, required_status_checks has to include include_admins field.